### ISSUE TITLE
Can't shim 'rg.exe': File doesn't exist.

### ISSUE THREAD

**Issue by mateusz-bajorek** [lukesampson/scoop#3880]
I've updated scoop and tried installing `ripgrep`, ended up with 
```shell
$ scoop install ripgrep
Installing 'ripgrep' (12.0.0) [64bit]
Loading ripgrep-12.0.0-x86_64-pc-windows-msvc.zip from cache
Checking hash of ripgrep-12.0.0-x86_64-pc-windows-msvc.zip ... ok.
Extracting ripgrep-12.0.0-x86_64-pc-windows-msvc.zip ... done.
Linking ~\scoop\apps\ripgrep\current => ~\scoop\apps\ripgrep\12.0.0
Creating shim for 'rg'.
Can't shim 'rg.exe': File doesn't exist.
```

I can see `rg.exe` in ` C:\Users\mateusz\scoop\apps\ripgrep\12.0.0\ripgrep-12.0.0-x86_64-pc-windows-msvc\` thou.

**Comment by Ash258** [comment]
Fixed in ScoopInstaller/Main#893

**Comment by mateusz-bajorek** [comment]
Thanks @Ash258 , correct - closing this.
